test/test_search.py

```diff
@@ -15,14 +16,39 @@ def objective(config, **kwargs):
 search_space = {"a": randint(0, 10), "b": randint(0, 100)}
 
 
+def param_bins(bin_n, bin_s, bin_t):
+    def params_estimator(config):
+        return config["a"]
+
+    bin_width = 10
+    min_config = {"a": 0}
+    max_config = {"a": bin_n * bin_width}
+
+    bins = ParamBins(
+        min_config,
+        max_config,
+        params_estimator,
+        num_bins=bin_n,
+        log_bins=False,
+        start_bin_size=bin_s,
+        empty_bin_tolerance=bin_t,
+    )
+    return bins, bin_width
+
+
 @pytest.mark.parametrize("search_strategy", methods)
 def test_multi_objective_search(search_strategy, num_samples=5):
+    bins, _ = (
```

Comment:
```suggestion
    bins, _ = (
        param_bins(10, 2, 1)
        if search_strategy == "stratified_random_search"
        else (None, None)
    )
```

Otherwise this throws an error